usecase/usecase-webapi-xwalk-tests/tests.android.xml

```diff
@@ -11,18 +11,39 @@
     <set name="Networking &amp;amp; Storage" background="brand-warning" icon="glyphicon-link" type="script">
       <testcase component="Crosswalk Use Cases/WebAPI" execution_type="manual" id="FileSystem" purpose="File System">
       </testcase>
+      <testcase component="sample" execution_type="auto" id="IndexedDB" purpose="IndexedDB">
```

Comment:
component="sample" come from demo-express, mean this sample come from demo-express,  is not usecase.